src/pages/settings/Wallet/Card/GetPhysicalCardAddress.tsx

```diff
@@ -37,11 +54,40 @@ function GetPhysicalCardAddress({
         FormActions.setDraftValues(ONYXKEYS.FORMS.GET_PHYSICAL_CARD_FORM, {country: countryFromUrl});
     }, [countryFromUrl]);
 
+    const handleAddressChange = useCallback((value: unknown, key: unknown) => {
+        const addressPart = value as string;
+        const addressPartKey = key as keyof Address;
+
+        if (addressPartKey !== 'country' && addressPartKey !== 'state' && addressPartKey !== 'city' && addressPartKey !== 'zipPostCode') {
```

Comment:
Please use the constants that we have for this here:

https://github.com/Expensify/App/blob/df69c80229acdc510064d2671e2fdde8095a93a7/src/types/form/HomeAddressForm.ts#L5C1-L12C12